src/main/java/frc/team2412/robot/Hardware.java

```diff
@@ -96,9 +95,6 @@
         // default address of TCA9548A
         public static final int I2C_MULTIPLEXER_ADDRESS = 0x70;
         public static final Port I2C_MULTIPLEXER_PORT = I2C.Port.kMXP;
```

Comment:
The multiplexer stuff can be removed as well since we're not directly connecting the multiplexer to the roboRIO.